### ISSUE TITLE
Region did not trigger

### ISSUE THREAD

**Issue by BigGeorgeTx** [owntracks/ios#718]
I am using OwnTracks iOS with the openHAB GPSTracker binding. I have using it for about a month and have loved it. It just worked all the time. 

Until last night, when we went out, several km from our home, and the region triggers didn’t do anything. There is nothing in the openHAB logs and I didn’t receive the notifications I usually receive on my iPhone or watch. I doubled checked that the app was in Move mode, and that the setting for location sharing is Always. 

I’m unsure what else to try to troubleshoot this.

**Comment by BigGeorgeTx** [comment]
I’m still experiencing seemingly random occurrences of a region trigger not firing. I have a series of regions around my home that trigger different actions: 1km, 500m, 250m and 75 m. Last night when we came home the 1k, 500 & 75 triggered as expected, but the 250 did not, though it did trigger a couple of hours later. (This is all as seen on openHAB through the openHAB.org mycloud server.

**Comment by lostfocus** [comment]
It seems to be an iOS thing as soon as the number of regions gets higher. I just added a 30th region a few days back and I start to see this behaviour. Sometimes it just ignores a region.

**Comment by ckrey** [comment]
Yes, this is an undocumented limitation in iOS. Don’t use 30 or more regions!

**Comment by ckrey** [comment]
Closing as answered and no more details from the original issue.
